cmd/gokey/main.go

```diff
@@ -100,6 +102,14 @@ func main() {
 	flag.Parse()
 
 	var err error
+	if pass == "" && passFile != "" {
+		var content []byte
+		content, err = ioutil.ReadFile(passFile)
+		if err != nil {
+			log.Fatalln(err)
+		}
+		pass = strings.TrimSpace(string(content[:]))
```

Comment:
should we delete a trailing newline as well?

Reply:
IMO we should delete trailing newlines which `strings.TrimSpace` does. For example, password files can be specified for rsync and the trailing newline is ignored there. Since editors might add newlines, we should not honor them.